### PR TITLE
Feature/home page

### PR DESCRIPTION
**Pull request by Mk996**

### CODE REVIEW

File: _templates/components/explore.html [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+    </p>
+    
+    <a class="btn-link explore-tile-know-more" href="https://dipy.org/documentation/1.7.0/interfaces/">
+      Know more
+    </a>
+    
+  </div>
+  <div class="explore-tile pad-h-20 pad-v-48">
+    <div class="explore-tile-icon">
+      <img src="_static/images/icons/cli.png" alt="" srcset="">
```
Comment (skoudoro) [inline _templates/components/explore.html:33]: Can you add and update all titles + icons

File: _templates/components/carousel.html [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    <button type="button" data-bs-target="#banners" data-bs-slide-to="2" aria-label="Slide 3"></button>
+  </div>
+  <div class="carousel-inner">
+    <div class="carousel-item active">
+      <img src="https://raw.githubusercontent.com/Garyfallidis/garyfallidis.github.com/master_bak/images/quickbundles_banner.jpg" class="d-block w-100" alt="...">
+      <div class="carousel-caption d-none d-md-block">
+        <h5>First slide label</h5>
+        <p>Some representative placeholder content for the first slide.</p>
+      </div>
+    </div>
```
Comment (skoudoro) [inline _templates/components/carousel.html:14]: I wonder it you should use a context here also, like you did for sponsor. it make more sense to me

File: _templates/components/explore.html [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+    </div>
+    <h4 class="explore-tile-title mar-v-5">Command-Line Interfaces</h4>
```
Comment (skoudoro) [inline _templates/components/explore.html:21]: In dark mode, the title is not enough emphasized. We do not see the bold effect. Maybe the text titles should be lighter in dark mode (closer to white)

**Review by skoudoro — COMMENTED** [review]
Hi @Mk996,

Thank you for this. See below my first comments. 

Also, do you plan to do the footer in this PR or on a new PR?

File: _templates/components/sponsors.html [hunk 4]
```diff
@@ -0,0 +1,8 @@
+<h2 class="section-title">Sponsors</h2>
+  <div class="grid-5 sponsors">
+    {% for sponsor in sponsors %}
+    <a href="{{ sponsor.url }}" class="sponsors-item" target="_blank">
+      <img src="_static/images/sponsors/{{ sponsor.image }}" alt="{{ sponsor.name }}">
+    </a>
+    {% endfor %}
+  </div>
```
[the remark this answers is not in the document]
Comment (Mk996) [reply _templates/components/sponsors.html:8]: desired, I saw it on the vue js

File: _templates/components/explore.html (hunk 3 above)
Comment (Mk996) [reply _templates/components/explore.html:21]: Let me work on that

File: _templates/components/explore.html (hunk 1 above)
Comment (Mk996) [reply _templates/components/explore.html:33]: Sure

File: context/sponsors.json [hunk 5]
```diff
@@ -0,0 +1,23 @@
+[
+  {
+    "name": "NIH NIBIB",
+    "image": "NIH_NIBIB.png",
```
[the remark this answers is not in the document]
Comment (Mk996) [reply context/sponsors.json:4]: No not currently, but I should not have background

**Comment by Mk996** [comment]
I am planning to create a new PR for footer

File: _templates/components/sponsors.html (hunk 4 above)
[the remark this answers is not in the document]
Comment (skoudoro) [reply _templates/components/sponsors.html:8]: ok fine with this feature

File: _templates/components/explore.html [hunk 6]
```diff
@@ -0,0 +1,129 @@
...
+    <p class="explore-title-description">
+      All the algorithms are available using CLI. You can also create your own algorithms.
+    </p>
+    
+    <a class="btn-link explore-tile-know-more" href="https://dipy.org/documentation/1.7.0/interfaces/">
+      Know more
+    </a>
+    
+  </div>
```
[the remark this answers is not in the document]
Comment (skoudoro) [reply _templates/components/explore.html:16]: So you need 12?   if we have 10, DO you think 5 in a row is ok ?

**Review by skoudoro — APPROVED** [review]
All good, thanks @Mk996, merging

**Pull request merged**